docs/data/data-grid/demo/demo.md

```diff
@@ -8,7 +8,7 @@ We're using the Data Grid to display the following features' table.
 Filtering, sorting, and grouping (by plan) are at your disposal to help you explore the list.
 :::
 
-{{"demo": "PopularFeaturesDemo.tsx", "defaultCodeOpen": false, "bg": "inline", "hideToolbar": true}}
+{{"demo": "PopularFeaturesDemo.js", "defaultCodeOpen": false, "bg": "inline", "hideToolbar": true}}
```

Comment:
These were interestingly wrong usages that used to work.
After bumping monorepo such imports failed to be parsed correctly.